link/src/main/java/com/stripe/android/link/account/LinkAccountManager.kt

```diff
@@ -32,57 +29,33 @@ internal class LinkAccountManager @Inject constructor(
     private val linkEventsReporter: LinkEventsReporter
 ) {
     private val _linkAccount = MutableStateFlow<LinkAccount?>(null)
-    var linkAccount: StateFlow<LinkAccount?> = _linkAccount
+    val linkAccount: StateFlow<LinkAccount?> = _linkAccount
+
+    @Volatile
+    @VisibleForTesting
+    var authSessionCookie: String? = null
 
     /**
      * The publishable key for the signed in Link account.
      */
+    @Volatile
+    @VisibleForTesting
```

Comment:
Nit: Can these be `internal`?

Reply:
The class is marked internal, so they're implicitly internal.

Reply:
Oh, didn’t see that in the GitHub snippet.